api/database/models.py

```diff
@@ -0,0 +1,96 @@
+from datetime import datetime, timezone
+from sqlalchemy import Column, String, Integer, Text, Boolean, DateTime
+from enum import Enum
+from api import db
+
+class Skill(db.Model):
+    __tablename__ = 'skills'
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(80), nullable=False)
+
+    created_at = Column(DateTime(timezone=True), nullable=False, default=datetime.now())
+
+    # Many to Many Relationship: nothing needed here.
+    applicants = db.relationship('Applicant', secondary='applicant_skills')
+
+class Value(db.Model):
+    __tablename__ = 'values'
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(80), nullable=False)
+
+    created_at = Column(DateTime(timezone=True), nullable=False, default=datetime.now())
+
+    # Many to Many Relationship: nothing needed here.
+    applicants = db.relationship('Applicant', secondary='applicant_values')
+
+class ApplicantSkill(db.Model):
+    __tablename__ = 'applicant_skills'
+    id = db.Column(db.Integer, primary_key=True)
+    applicant_id = db.Column(db.Integer, db.ForeignKey('applicants.id'))
+    skill_id = db.Column(db.Integer, db.ForeignKey('skills.id'))
+
+    created_at = Column(DateTime(timezone=True), nullable=False, default=datetime.now())
+
+    # relationships
+    applicant = db.relationship('Applicant', backref=db.backref("applicant_skills", cascade="all, delete-orphan"))
+    skill = db.relationship('Skill', backref=db.backref("applicant_skills", cascade="all, delete-orphan"))
+
+class ApplicantValue(db.Model):
+    __tablename__ = 'applicant_values'
+    id = db.Column(db.Integer, primary_key=True)
+    applicant_id = db.Column(db.Integer, db.ForeignKey('applicants.id'))
+    value_id = db.Column(db.Integer, db.ForeignKey('values.id'))
+
+    created_at = Column(DateTime(timezone=True), nullable=False, default=datetime.now())
+
+    # relationships
+    applicant = db.relationship('Applicant', backref=db.backref("applicant_values", cascade="all, delete-orphan"))
+    value = db.relationship('Value', backref=db.backref("applicant_values", cascade="all, delete-orphan"))
+
+class Applicant(db.Model):
+    __tablename__ = 'applicants'
+
+    # The name of the column is the name you assign it to
+    id = Column(Integer, primary_key=True)
+    email = Column(String(100), unique=True, nullable=False)
+    first_name = Column(db.String(80))
+    last_name = Column(String(80))
+
```

Comment:
What did we decide with @HireUp-Turing/front-end regarding how an applicant's name appears, if at all? Is that a column we need to include in the Applicants table?

Reply:
we didn't really decide. I wrote these columns based on the dbschema that you had created (the image I put in the readme for reference). We can always change this to suit later.